data/plugins/thirdparty/matrix-to-discourse.yaml

```
@@ -0,0 +1,7 @@
+name: Matrix To Discourse
+repo: https://github.com/gitayam/matrix-to-discourse
+license: GPL 3.0
```

The license field both here and in `maubot.yaml` should be a valid SPDX license identifier (https://spdx.org/licenses/)

```suggestion
license: GPL-3.0-or-later
```

or

```suggestion
license: GPL-3.0-only
```

---

fixed it!